src/solvers.jl

```diff
@@ -250,6 +250,8 @@ function solve(m::Model; suppress_warnings=false,
             # Do a separate try since getobjval could work while getobjbound does not and vice versa
             objBound = MathProgBase.getobjbound(m.internalModel) + m.obj.aff.constant
             m.objBound = objBound
+        catch
+            nothing
```

Comment:
Why is this needed?

Reply:
On v0.7 each try block needs either a ``catch`` or a ``finally`` condition